src/app/shared/player/player.component.ts

```diff
@@ -268,7 +274,7 @@ export class PlayerComponent implements OnInit, OnChanges {
 
   togglePlayPause() {
     if (this.player.paused) {
-      this.player.play();
+      this.player.play().then(_ => this.sessionService.playbackStarted());
```

Comment:
It's all pulled automatically by the browser from whether there is an audio element playing on the page. We put this in place to not show the notification until audio playback has started, so if there's an error a notification doesn't show up anyway.
The API for mediasession is literally just metadata and callbacks for transport controls, so you can't even report a play state.